src/Pages/Auth/Register/index.js

```diff
@@ -1,15 +1,113 @@
-import React from "react"
+import React, { useState } from "react"
+import { useQuery } from "react-query"
 import { Link } from "react-router-dom"
+import Input from "components/Input"
+import Button from "components/Button"
+import { useI18n } from "providers/I18n"
 
 Register.propTypes = {}
 
 function Register() {
+  const { t } = useI18n()
+
+  const [urlUserCode, setUrlUserCode] = useState(window.location.href)
+
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+    firstName: "",
+    lastName: "",
+    dob: "",
+    profession: "",
+    passwordConfirmed: ""
+  })
+
+  const getUserData = async () => {
```

Comment:
This function can be extracted out of the component. Let's copy this function in `src/api/auth.js` and change its signature so that it accepts a string variable for the code

Reply:
you mean to remove it and manually import it when i need to use it?

Reply:
Yes. His function will live in src/api/auth.js and we will import it here.

Reply:
Should i create the path src/api/auth.js ? Because I think it doesn't exist

Reply:
yes